Azure/sap-hana
# Change ansible file location to TF workspace

## Problem
Current files used for ansible are saved under `sap-hana/deploy/v2` which is a problem for multiple deployments using different workspaces (#491).

## Description
With this PR:
1. Change file location under workspace.
1. Trigger file transfer when file content changes (eg. change `high_availability` from `false` to `true`)
1. Modify the pipeline accordingly.

## Test
1. Deploy with default workspace and `high_availability: false`.
1. Make sure files are generated under `sap-hana/default`.
1. Confirm that the correct files are copied over to RTI.
1. Set `high_availability` to `true`, deploy, then confirm the correct files are transferred to RTI.
